server/src/main/java/org/elasticsearch/common/logging/LogConfigurator.java

```diff
@@ -119,6 +120,13 @@ public static void configure(final Environment environment) throws IOException,
         configure(environment.settings(), environment.configFile(), environment.logsFile());
     }
 
+    /**
+     * Load logging plugins so we can have {@code node_name} in the pattern.
+     */
+    public static void loadPlugins() {
```

Comment:
Can you name this something like "loadLog4jPlugins" so it is not confused with ES plugins (which it took me a few minutes to realize when reading this PR)?

Reply:
++